config/custom-environment-variables.yml

```diff
@@ -1,22 +1,19 @@
-node:
-  config:
-    env: 'NODE_CONFIG_ENV'
+accounts:
+  tokenKey: 'JWT_TOKEN'
```

Comment:
`secret` under jwt was better imo. Also could we introduce refresh tokens as well into the boiler?

Reply:
> secret under jwt was better imo
@vjnvisakh-jtc - By `secret`, do you mean `jwt.token` ? If yes - `jwt.token` configuration is too generic and I have seen developers using this config for every JWT that they sign, which is not a right thing to do. Any specific reason why was it better?

> Also could we introduce refresh tokens as well into the boiler?
That is a planned item and it out of the scope of this PR.

Reply:
I always thought `secret` as something used for signing and `token` as the end result. Okay with me.